src/main/resources/templates/generators/overviewFeatures.vm

```diff
@@ -60,10 +60,11 @@
       </div>
     </div>
   </div>
-  <br/>
+#else
```

Comment:
Will be happy to have tests that covers lack of chart. Not sure if we have test that validates test availability, please check

Reply:
Not sure what you are specifically looking for here. The tests added to ConfigurationTest verify the Set/List logic.

Reply:
This tests verify that the flag is added to configuration but not if the chart is really rendered or not

Reply:
Correct, I'm not sure how to go about validating chart is rendered or not within this project.